pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,349 @@
+
+This allows brands to collect large amounts of information from customers using a seamless interface that is contained within the Apple Messaging Interface. This can allow for a wide range of interactivity without requiring the consumer to leave the conversation.
+
+Sending the structured content templates (body and metadata) explained in this document will allow you to send Business Forms to the consumer device. 
+
+For instructions on how to send structured content with metadata in Conversational Builder, follow the [steps](https://developers.liveperson.com/conversation-builder-interactions-code.html#updating-conversation-metadata)
+
+Below is an image of a Business Forms:
+
+<img  style="width:350px" src="img/form_message_1.1.png" alt="Splash screen">
+<img  style="width:350px" src="img/form_message_1.2.png" alt="Select page">
+<img  style="width:350px" src="img/form_message_1.3.png" alt="Response page">
+
+### Pages
+Following Types of pages are supported
+
+- **splash**: An introduction page that includes a title, description, action button and an image.
+
+- **select**: A page that features either a single “True/False” selection or multiple select. The page also supports image assets with each selection.
+
+- **picker**: A page that features a single select from multiple options.
+
+- **datePicker**: Provides Apple’s standard Date Picker with the ability to change the date format and set maximum/minimum dates.
+
+- **input**:  A page that can feature various input fields to collect strings of information.
+
+**Every page has following properties**
+
+|Property Name|Description|Type|Required|
+|--- |--- |--- |--- |
+|pageIdentifier|A unique identifier, as a string, for the page being shown, that is less than 20 characters in length. Do not reuse. Ex: "firstNamePage" or "0001".|String|Y|
+|type|Defines Type of page. Use one of the following values: select, picker, datePicker, or input.|String|Y|
+|title|If set this displays the title in bold on the page.|String|N|
+|subtitle|Use this object to display the question for the page.|String|Y|
+|nextPageIdentifier|except for single select option page, where you specify the nextPageIdentifier within each of the item objects. If a page does not have a nextPageIdentifier set on it, the client assumes the current page to be the final page and proceeds to send or provides a summary page based on the data. This is a unique page identifier to show the next page.|String|Y|
+|submitForm|A Bool value placed on the pages to denote the end page of the form. Since multiple pages can act as an end page, this object can be set on multiple pages.(must be provided on last page)|Bool|N|
+|options|Dictionary containing optional values for the input field. Some of which are required, keyboardType,inputType, maximumCharacterCount etc.|JSON|Y|
+
+  **Page Specific Properties**
```

Comment:
Please “unindent” these two lines, and add a new line between “heading” and table.

Reply:
@j9t what do you mean by unindent, i not sure what needs to be done.